README.md

```
@@ -1,304 +1,424 @@
-```
-npm install  --no-optional --save-dev react-responsive@^5.0.0
-```
+**Q: What about projects with cloud variables? Can I still play online games with OmniBlocks?**
+A: Yup! The TurboWarp cloud data server is used to ensure that you can still play your favorite online games in OmniBlocks. Keep in mind that since we don't integrate with Scratch, you can choose any username that exists in Scratch (except for Scratch Team member names), so if you see someone claiming to be a famous Scratcher (e.g., griffpatch), it is most likely not actually that person.
 
-Further reading: [Stack Overflow](https://stackoverflow.com/questions/46602286/npm-requires-a-peer-of-but-all-peers-are-in-package-json-and-node-modules)
+### Features & Development
 
-## Troubleshooting
+**Q: What's the status of the music editor feature?**  
+A: The music editor is currently in development and not fully integrated. We're working on it, but for now, the traditional sound editor is available. You can go ahead and use it, but if you click out of the tab... well, say goodbye to your music. You can test the raw song editor [here](https://omniblocks.github.io/songeditor.html)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)